acr-build.yml

```diff
@@ -14,13 +14,23 @@ steps:
       .
     keep: true
 
+  - id: divorce
+    build: >
+      -t {{.Run.Registry}}/ccd/logstash:divorce-{{.Run.ID}} -t {{.Run.Registry}}/ccd/logstash:divorce-latest
+      --build-arg conf=ccddivorce_logstash.conf.in
+      .
+    keep: true
+
   - id: push-images
     push:
       - "{{.Run.Registry}}/ccd/logstash:latest"
       - "{{.Run.Registry}}/ccd/logstash:ccd-latest"
       - "{{.Run.Registry}}/ccd/logstash:ccd-{{.Run.ID}}"
       - "{{.Run.Registry}}/ccd/logstash:probate-latest"
       - "{{.Run.Registry}}/ccd/logstash:probate-{{.Run.ID}}"
+      - "{{.Run.Registry}}/ccd/logstash:divorce-latest"
+      - "{{.Run.Registry}}/ccd/logstash:divorce-{{.Run.ID}}"
```

Comment:
why you need this?
The ccd divorce LS is already published under `ccdprobate-logstash`

Reply:
I'm not sure how @linusnorton's commit got in my PR :D

Reply:
that should have been done as part of https://tools.hmcts.net/jira/browse/RDM-10064
let me check hold on

Reply:
ignore this comment, I thought we already had modified this file to publish the various ccd LS instances in ACR as part of https://tools.hmcts.net/jira/browse/RDM-10064 as was requested in that ticket, but I found out that wasn't done instead

Reply:
but yes Linus apparently has already done the change for Divorce, but we should call that `ccddivorce-logstash`